driver-examples/tycho/README.md

```diff
@@ -0,0 +1,51 @@
+# Java Driver examples
+
+This project demonstrates how to integrate the Java Driver in an Eclipse RCP application
+built with Maven and Tycho.
+
+## Packaging the Java Driver in an Eclipse RCP application
+
+The Java Driver is built and distributed as an OSGi bundle, but integrating it in an RCP application is not trivial.
+
+If your project/company benefits from a dedicated P2 update site, it is best to place the driver bundle, 
+along with its dependencies, in your update site, and have Tycho resolve the dependencies from there.
+
+Alternatively, it is also possible to "wrap" the Java Driver jar, along with its dependencies,
+in a plugin submodule in your application, and have Tycho build it along with your application.
+
+The current project uses a third option: using the [p2-maven-plugin](http://projects.reficio.org/p2-maven-plugin),
+it is possible to build a local P2 installation directory, then use Jetty to serve it.
+
+## Creating the local P2 installation
```

Comment:
Introducing this new plugin, p2-maven-plugin, gives me more flexibility but the build is more complex now, and goes through two phases: first, build the P2 site, then build the application.